python/ray/autoscaler/kuberay/kuberay-autoscaler.yaml

```diff
@@ -1,70 +0,0 @@
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: autoscaler-config
-data: 
-  ray_bootstrap_config.yaml: |
-    cluster_name: raycluster-complete
-    min_workers: 0
-    max_workers: 10
-    idle_timeout_minutes: 1
-    provider:
-      type: kuberay
-      namespace: default
-      disable_node_updaters: True
-      disable_launch_config_check: True
-    head_node_type: head-group
-    available_node_types:
-      head-group:
-        resources: {"CPU": 1}
-        node_config: {}
-        max_workers: 1
-      small-group:
-        resources: {"CPU": 1}
-        node_config: {}
-        min_workers: 0
-        max_workers: 10
-    file_mounts: {}
-    cluster_synced_files: []
-    file_mounts_sync_continuously: False
-    initialization_commands: []
-    setup_commands: []
-    head_setup_commands: []
-    worker_setup_commands: []
-    head_start_ray_commands: []
-    worker_start_ray_commands: []
-    auth: {}
-    head_node: {}
-    worker_nodes: {}
----
-apiVersion: v1
```

Comment:
This is important: The service account, role and role binding definitions are still needed.

Related, what is the plan for doing an end-to-end test of the autoscaler in the Ray CI? Do we already have a PR for that? It would have caught this problem.

Reply:
Thanks for catching that! Only meant to remove the configMap.

Plan: I'll write tests relatively soon that can be run against any K8s cluster.
@simon-mo is working on the CI infra to automated with minikube with a target of ~3/15.